packages/web-runtime/src/components/UploadInfo.vue

```diff
@@ -356,7 +356,7 @@ export default defineComponent({
         this.handleTopLevelFolderUpdate(file, 'error')
       }
     })
-    this.$uppyService.subscribe('uploadSuccess', (file) => {
+    this.$uppyService.subscribe('uploadSuccess', (file: any) => {
```

Comment:
Should be `UppyResource` here I believe.

Reply:
Ah, yes - the type was just missing `isFolder` property.